src/tango/quic/tests/arp/test_quic_arp_server.c

```diff
@@ -128,8 +128,6 @@ main( int argc, char ** argv ) {
     .stream_cnt       = { 2, 2, 2, 2 },
     .inflight_pkt_cnt = 1024,
     .tx_buf_sz        = 1<<14,
-    .routing_entries  = 32,
-    .arp_entries      = 32
```

Comment:
Is there a reason to get rid of these and only use hardcoded 256?

Reply:
The caller configures them now, not QUIC. This caller could set 32 if it wanted.